migration/template/apply/20-bootstrap.sh

```diff
@@ -6,10 +6,16 @@ ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
 source "${ROOT}/scripts/migration/lib.sh"
 
 run() {
+  local cluster="${2:-both}"
```

Comment:
Use the environment variable instead of an argument if you go with an environment variable.